sequence/sequence_manager.py

```diff
@@ -0,0 +1,29 @@
+from sequence.sequence_factory import get_sequence_generator
+
+TABLE_NAME = 'generated_id_counter'
+
+
+class IdGenerator:
+    """A component that provides sequential ids"""
```

Comment:
Could create unit tests for each of these methods and assert mock has been called on sequence generator.

Reply:
I don't think it is necessary, the methods are tested in component_test_ids_generators.

Reply:
I think unit test could be useful here instead of relying on component tests. @mattd-kainos thoughts?

Reply:
OK, @Broom94 , I have just added the unit test :) Please review it :)